apps/astarte_realm_management/lib/astarte_realm_management/queries.ex

```diff
@@ -704,37 +668,22 @@ defmodule Astarte.RealmManagement.Queries do
   end
 
   defp delete_all_paths(
-         client,
+         realm_name,
          device_id,
          %InterfaceDescriptor{
            interface_id: interface_id,
            storage_type: :multi_interface_individual_datastream_dbtable
          } = _interface_descriptor
        ) do
-    delete_paths_statement = """
-    DELETE
-    FROM individual_properties
-    WHERE device_id=:device_id AND interface_id=:interface_id
-    """
+    keyspace = Realm.keyspace_name(realm_name)
 
-    all_paths_query =
-      DatabaseQuery.new()
-      |> DatabaseQuery.statement(delete_paths_statement)
-      |> DatabaseQuery.put(:device_id, device_id)
-      |> DatabaseQuery.put(:interface_id, interface_id)
-      |> DatabaseQuery.consistency(:each_quorum)
+    query =
+      from IndividualProperty,
+        where: [device_id: ^device_id, interface_id: ^interface_id]
 
-    with {:ok, _result} <- DatabaseQuery.call(client, all_paths_query) do
-      :ok
-    else
-      {:error, reason} ->
-        _ =
-          Logger.warning("Database error while deleting all paths: #{inspect(reason)}.",
-            tag: "db_error"
-          )
+    _ = Repo.delete_all(query, consistency: :quorum, prefix: keyspace)
```

Comment:
Previously, the query consistency was `each_quorum`.

```suggestion
    _ = Repo.delete_all(query, consistency: :each_quorum, prefix: keyspace)
```